### PR TITLE
Task 11 PR

### PR DESCRIPTION
**Pull request by sachidanandaEpam**

### CODE REVIEW

**Comment by sachidanandaEpam** [comment]
image
image
image
image
image

File: src/app/core/services/validation.service.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ValidationService {
+
+  constructor() { }
+
+  public buildFormGroup(items: FormFieldItem[]): FormGroup {
+    const group: any = {};
```
Comment (MariMax) [inline src/app/core/services/validation.service.ts]: it should be `Record<string, FormControl>`

File: src/app/shared/components/users-picker/users-picker.component.ts [hunk 2]
```diff
@@ -16,35 +17,37 @@ import { UserService } from 'src/app/core/services/user.service';
...
   }
 
-  @Input() public formFieldItem: FormFieldItem;
-
-  private disabled = false;
-  private _value: Author[];
-  private _onChange: (value: any) => void = () => { };
-  private _onTouched = () => { };
+  constructor(private _user: UserService) {
+    console.log('inside users picker component constructor');
```
Comment (MariMax) [inline src/app/shared/components/users-picker/users-picker.component.ts]: something you should not do, there is a rule in eslint no-console, so in theory it should warn you

File: src/app/core/services/validation.service.ts (hunk 1 above)
Comment (sachidanandaEpam) [reply src/app/core/services/validation.service.ts]: Updated

File: src/app/shared/components/users-picker/users-picker.component.ts (hunk 2 above)
Comment (sachidanandaEpam) [reply src/app/shared/components/users-picker/users-picker.component.ts]: fixed.

**Pull request merged**